pkg/gitparse/gitparse_test.go

```diff
@@ -754,6 +755,12 @@ func TestIndividualCommitParsing(t *testing.T) {
 	}
 }
 
+func newBufferedFileWriterWithContent(content []byte) *bufferedfilewriter.BufferedFileWriter {
+	b := bufferedfilewriter.New()
+	_, _ = b.Write(context.Background(), content) // Using Write method to add content
```

Comment:
Do you think it's worth panicking or something if this fails? I can imagine it being really annoying for this to silently fail during a test.

Reply:
Oh so i added this back specifically to test the bufferedfilewriter implementation of the contentWriter. For example the `TestStagedDiffParsingBufferedFileWriter` uses it instead of the default `buffer` contentWriter.